src/osd/ReplicatedPG.cc

```diff
@@ -8579,6 +8579,7 @@ void ReplicatedPG::simple_opc_submit(OpContextUPtr ctx)
   dout(20) << __func__ << " " << repop << dendl;
   issue_repop(repop, ctx.get());
   eval_repop(repop);
```

Comment:
some functions call simple_opc_submit more than one time, is it the right place without redundancy.

Reply:
in the cases where it's called in a loop, it shouldn't be too expensive - it'll either bail out early since limit == pg_trim_to, or have to iterate through a small number of log entries (rougly osd_pg_log_trim_min entries)